docs/en/Go-Client.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: global
+title: Go Client
+nickname: Go Client
+group: Clients
+priority: 3
+---
+
+Alluxio has a [Go Client](https://github.com/Alluxio/alluxio-go) for interacting with Alluxio through it's 
+[REST API](Clients-Rest.html). It exposes an API very similar to the native Java API. The Go client requires 
+that an Alluxio proxy service is running. See the [godoc](http://godoc.org/github.com/Alluxio/alluxio-go) 
```

Comment:
NIT: If we are explicit that the proxy is required, should we also mention why (go client uses the REST api)?